.github/ISSUE_TEMPLATE/user-feedback.yml

```diff
@@ -11,6 +11,22 @@ body:
   - type: markdown
     attributes:
       value: Thank you for offering feedback! This form is intended for general user impressions and reactions to the Qiskit and IBM Quantum documentation. If you would like to report a bug or typo, request new content, or tell us about a problem with the UI, please return to the <a href="https://github.com/Qiskit/documentation/issues/new/choose">issues</a> page and select one of the other issue types. If you have questions about this form or more information to share, please reach out to <a href="mailto:quantum.design@us.ibm.com">quantum.design@us.ibm.com</a>.
+  - type: markdown
```

Comment:
I'm not certain this one makes sense either. I want to make sure that our team is aware of overall user feedback.